src/pages/patterns/buttons.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Buttons
+description: >
+  TODO BEED TO FIGURE OUT DESCRIPTION
```

Comment:
@allisonacs, @ctancinco This section also needs to copy help.

Reply:
Looks fine to me. 

Haha, I'll review these today.

Reply:
You have an existing description on the site now: “Buttons let users take action quickly. Use a Button to enable actions or decisions that are important to a user's workflow.” If you don't want to reuse that, we could just edit this down to: "Buttons highlight and enable an important action or decision within a workflow." LMK if you wanted to message something differently.

Reply:
@jzempel @allisonacs what is your decision on this?

Reply:
@ctancinco do we need to differentiate the Component overview from the pattern overview?

Patterns are how you put these components together, how you use them properly… that certainly overlaps with what we have/your suggestion, but doesn't imply that the article tells you how to do that / tells you best practices. 

I'm also mindful that these will all be housed in the "patterns" section itself, so battering people with phrasing like "and here are the best practices for that" will be repetitive and maybe not necessary. 

What do you think?

Reply:
@allisonacs Sorry, I didn't see we were talking pattern vs component. Yes, it should be treated differently than components but it has to be shorter than the pattern overview. 

Option 1: Thesis statement. Distill the philosophy behind button design into one line. Examples: 

- There are different buttons for different needs
- Understand the intention behind the action
- Know why they're pushing your buttons

Option 2: best practices/how-to but, as you mentioned, will get repetitive once you add more patterns:

- Tips for better button design
- How to design better buttons

LMK if any of these are closer to the mark.

Reply:
> Know why they're pushing your buttons

I love that. 

When we look at the component description overviews, they've got a period at the end, hopefully we can do that here too.

Reply:
absolutely: "Know why they're pushing your buttons."

Reply:
Updated. Thanks for the help.